src/core/services/layout/layouts.js

```diff
@@ -0,0 +1,108 @@
+(function () {
+
+  'use strict';
+
+  angular.module('material.layouts', ['material.core'])
```

Comment:
Add this comment header to line 4:

```js
/**
 *
 *   The original ngMaterial Layout solution used attribute selectors and CSS.
 *  
 *  ```html
 *  <div layout="colum"> My Content </div>
 *  ```
 *  
 *  ```css
 *  [layout] {
 *    box-sizing: border-box;
 *    display:flex;  
 *  }
 *  [layout=column] {
 *    flex-direction : column
 *  }
 *  ```
 *  
 *  Use of attribute selectors create significant  performance impacts in some browsers...
 *  mainly IE. 
 *  
 *  This module registers directives that allow the same layout attributes to be intrepreted 
 *  and converted to class selectors:
 *  
 *  ```css
 *  .layout {
 *    box-sizing: border-box;
 *    display:flex;  
 *  }
 *  .layout-column {
 *    flex-direction : column
 *  }
 *  ```
 */
```